staff/NuriaComa/cv/index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Currículum Vitae</title>
+</head>
+<body>
+    <h1> CV_Núria Coma</h1>
+    <h2>estudios</h2>
+    <ul>
+        <li>Arquitectura superior ETSAB</li>
+        
+    </ul>
+    <h2>Trabajos</h2>
+        <h3>Arquitectura</h3>
+            <p>Trabajos en despatxos de arquitectura como arquitecta</p>
+        <h3>Interiorismo</h3>
+            <p>Trabajos en despatxos de Interiorismo como arquitecta</p>
+    <h2>Idiomas</h2>
+        <ul>
+            <li>Català</li>
+            <li>Castellano</li>
+            <li>English</li>
+        </ul>
+    <h2>Premios</h2>
+        <ul>
+            <li> <h4>Premi FAD d'arquitectura 2013 </h4></li>
```

Comment:
WARN! do not nest H4 inside LI! you can try using H4 here, without UL/LI, which are already used before.

Reply:
Done!